spinalcordtoolbox/download.py

```diff
@@ -195,3 +349,18 @@ def install_data(url, dest_folder, keep=False):
     logger.info("Removing temporary folders...")
     shutil.rmtree(os.path.dirname(tmp_file))
     shutil.rmtree(extraction_folder)
+
+
+def install_named_dataset(dataset_name, dest_folder=None, keep=False):
+    """
+    A light wrapper for the 'install_data' function to allow downstream consumers to download
+    datasets using only the dataset's name (i.e. without needing to access DATASET_DICT fields).
+    """
+    if dataset_name not in DATASET_DICT.keys():
+        raise ValueError(f"{dataset_name} is not contained in list of datasets. Choose from: {DATASET_DICT.keys()}")
```

Comment:
A perhaps more readable error:

```suggestion
        raise ValueError(f"{dataset_name} is not contained in list of datasets. Choose from:\n\n {pprint.pformat(set(DATASET_DICT.keys()))}")
```

Reply:
Ah, good catch! I didn't think to check the output of printing a `keys` object as-is:

Before:

![image](https://user-images.githubusercontent.com/16181459/195861114-7f464850-9c86-4c63-828c-faafd72b40bb.png)

After:

![image](https://user-images.githubusercontent.com/16181459/195861386-e92e1ded-16e6-4301-91ac-354ba8a6feaf.png)

Reply:
It doesn't look as pretty as I was hoping and `dict_keys()` isn't as distracting as I thought it might be.

This is good to merge either way.

Reply:
As a compromise, I think I might just use `list(DATASET_DICT.keys())` to remove the `dict_keys` bit but preserve the condensed list.

![image](https://user-images.githubusercontent.com/16181459/196451105-2df9bba8-8e5d-4c4d-81f5-d3873d049af3.png)

Reply:
Oh! TIL that sorting in Python treats capital letters as a different set than lowercase letters. We get the same sorting even using `sorted()`.

Reply:
Neat. In Python 3.3+, you can use [`key=str.casefold`](https://stackoverflow.com/a/10269828) to take care of this.